pkg/controller/velerobackup/velerobackup_controller.go

```diff
@@ -36,6 +38,9 @@ const (
 )
 
 var (
+	// Default reconcile rate limiting to 1 backup every 3 minutes.
+	reconcileRateLimitDuration = 180 * time.Second
```

Comment:
Make this a constant and store the variable to be used by the reconciler in a field of `ReconcileBackup`.